float32 SensorL
float32 SensorC
float32 SensorR
